tests/XRootD/scitokens.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+function setup_scitokens() {
+	require_commands curl
+	if [ -z "$HOSTNAME" ]; then
+		HOSTNAME=`hostname`
+	fi
+
+	cat > "$PWD/scitokens/authdb" << EOF
+u * /issuer lr
+EOF
+	cat > "$PWD/scitokens/scitokens.cfg" <<EOF
+[Global]
+audience = https://$HOSTNAME:8095
+
+[Issuer test]
+issuer = https://$HOSTNAME:8095/issuer/one
+base_path = /protected
```

Comment:
This file could also be made constant by using `https://localhost:8095/issuer/one` instead of `$HOSTNAME`.

Reply:
Done.